README.md

```diff
@@ -6,23 +6,50 @@
 
 ## Development setup
 
-Windows, OS X & Linux::
+Windows, OS X & Linux:
+
+
+## Requirements:
+
+You need to have Docker and Docker-compose installed on your machine.
+
+## Run project
+
+For run docker and enter into the container typing in a terminal the following commands.
 
-- Run docker container & install dependencies
 ```sh
 docker-compose up -d
 docker-compose exec -u web web bash
-./bin/console doctrine:migra:migra
 ```
 
-- Build assets
-```sh
-docker-compose -f docker-compose.cli.yml run --rm yarn install
-docker-compose -f docker-compose.cli.yml run --rm yarn build
+To install the necessary dependenceies of the project, both for php
+and javascript, inside the container execute the following commands.
+
+```bash
+composer install -o
+yarn install
 ```
 
-- Go to http://localhost:8080/
+If is your first time running this project, you must type the
+following commands to migrate to database, remember to run this
+command every time to added changes to database.
+
+```bash
+bin/console doctrine:migra:migra
+```
+
+For build assets, you only need run the following command, remember
+run this command every time when you add changes to styles files or
+javascript files.
+
+```sh
+yarn install
+yarn build
+```
 
+To see the portal of phpmx, write the address `http://localhost:8080/` 
```

Comment:
```suggestion
To see the portal of phpmx, write the address [http://localhost:8080/](http://localhost:8080/) 
```

Reply:
Being a free general purpose manual for anyone who can read it in a repository, putting a link that is only active in local mode and as long as the project is executed, I do not see the need for this change in the documentation.